appservice/src/main/java/nu/yona/server/BruteForceAttemptService.java

```diff
@@ -1,136 +0,0 @@
-/*******************************************************************************
```

Comment:
Hmm... Now I wonder what happened to the brute force checking on new device request? Is there any brute force checking done there still?